R/utils.R

```diff
@@ -10,6 +10,10 @@ get_csrf_token = function(servername, username, password) {
   index_res = GET(index_url)
   csrf_token = httr::headers(index_res)$`x-csrf-token`
 
+  if (is.null(csrf_token)) {
```

Comment:
Is this the correct error condition we're looking for? Did I add this? We actually want to do a check on user_type to ensure that it's paid_user vs anonymous user.

Reply:
I added it. In my testing on the rsurveycto server, if the server doesn't exist or isn't running, the csrf token will be NULL. If the server is running and I give it my correct username and password, the user_type is community_user. I'll dig into it more.